### PR TITLE
feat: added authentication pages

### PR DESCRIPTION
**Pull request by suryxks**
Added authentication pages

- Login
- signup

### CODE REVIEW

**Review by msdeshmukh009 — APPROVED** [review]
Good to go!!

File: pages/authentication.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+.form-grp>button{
+    margin: 1rem auto;
+}
+.form-grp>a{
+    text-decoration: none;
+}
+.form-accept{
+    display: flex;
+    justify-content: space-between;
```
Comment (Devansu-Yadav) [inline pages/authentication.css:29]: This CSS style rule is being applied on your Signup page (https://deploy-preview-4--nectar-decor.netlify.app/pages/signup.html) and hence giving too much space between the checkbox and the `accept terms and conditions` text. Please fix this minor issue.

**Review by Devansu-Yadav — APPROVED** [review]
Looks good to merge.

File: pages/authentication.css (hunk 1 above)
Comment (suryxks) [reply pages/authentication.css:29]: Thank you for pointing it out Devanshu I have made the changes and will merge the pr

**Pull request merged**